libs/tree-sitter-wing/package.json

```diff
@@ -18,10 +18,7 @@
   "dependencies": {
     "nan": "^2.12.1"
   },
-  "devDependencies": {
-    "tree-sitter-cli": "^0.20.6"
-  },
   "scripts": {
-    "test": "tree-sitter test"
+    "test": "npx -y tree-sitter-cli@0.20.7 test"
   }
```

Comment:
why not as a normal devDep?

Reply:
running npm install for this package runs node-gyp, which is both slow and caused issues in the build because it's expected that the c output had already been generated

I just realized I can add `--ignore-scripts` to fix that though so I'll add it back.